resources/aws/vpc.go

```diff
@@ -128,9 +132,16 @@ func (v *VPC) Delete() error {
 		return microerror.Mask(err)
 	}
 
-	if _, err := v.Clients.EC2.DeleteVpc(&ec2.DeleteVpcInput{
-		VpcId: vpc.VpcId,
-	}); err != nil {
+	deleteOperation := func() error {
+		if _, err := v.Clients.EC2.DeleteVpc(&ec2.DeleteVpcInput{
+			VpcId: vpc.VpcId,
+		}); err != nil {
+			return microerror.Mask(err)
+		}
+		return nil
+	}
```

Comment:
During testing I had problems with VPCs not being deleted due to other resources that were not yet deleted. So I've added retry logic.